src/backend/legendary/games.ts

```diff
@@ -643,6 +643,27 @@ class LegendaryGame extends Game {
           : ['--wine', wineBin])
       )
     }
+
+    // Log any launch information configured in Legendary's config.ini
+    await runLegendaryCommand(['launch', this.appName, '--json'], {
```

Comment:
Capturing only the `stdout` of Legendary (with `const { stdout } = runLegendaryCommand(...)`) will spare you from doing that output matching (you can then just pipe that into `JSON.parse` directly

Reply:
thanks! updated